api/src/resolvers/campaigns/campaigns-resolver.ts

```diff
@@ -6,14 +7,86 @@ import {
   createFellowshipEquipmentRecord,
   SetEquipmentDeadline,
   getEquipmentCampaign,
+  getConstituencyOverseersEmailsAndNumbers,
+  getFellowshipLeadersEmailsAndNumbers,
+  getEquipmentCampaignDate,
 } from './campaigns-cypher'
 
+import texts from '../texts.json'
+
 import { isAuth, rearrangeCypherObject, throwErrorMsg } from '../utils/utils'
 import { Context } from '../utils/neo4j-types'
 import { ChurchLevel } from '../utils/types'
+import { sendBulkEmail, sendBulkSMS } from '../utils/notify'
 
 const campaignsCypher = require('./campaigns-cypher')
 
+type Record = { keys: number[]; _fields: never[] }
+
+const sendEmailsandSMS = async (
+  args: { startDate: Date; endDate: Date; id: string; target: number },
+  context: Context
+) => {
+  const session = context.executionContext.session()
+
+  const overseersPhoneNumbers: string[] = []
+  const overseersEmailAdresses: string[] = []
+  const fellowshipPhoneNumbers: string[] = []
+  const fellowshipEmailAdresses: string[] = []
+
+  const constituencyLeadersResponse = await session.run(
+    getConstituencyOverseersEmailsAndNumbers,
+    args
+  )
+
+  const fellowshipLeadersResponse = await session.run(
+    getFellowshipLeadersEmailsAndNumbers,
+    args
+  )
+
+  constituencyLeadersResponse.records.forEach((record: Record) => {
+    overseersPhoneNumbers.push(record._fields[2])
+    overseersEmailAdresses.push(record._fields[1])
+  })
+
+  fellowshipLeadersResponse.records.forEach((record: Record) => {
+    fellowshipPhoneNumbers.push(record._fields[2])
+    fellowshipEmailAdresses.push(record._fields[1])
+  })
+
+  const equipmentDeadline = new Date(args.endDate)
+  const formattedDeadline = equipmentDeadline.toLocaleString('en-GB', {
+    weekday: 'long',
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+    timeZone: 'UTC',
+  })
+
+  await Promise.all([
+    sendBulkSMS(fellowshipPhoneNumbers, texts.equipment.fellowship),
```

Comment:
and these names leave something to be desired in terms of explaining what kind of text messages are being sent out. 
Change it to `notify_fellowship_leaders` or something